packages/react-devtools-shared/src/utils.js

```diff
@@ -260,9 +260,7 @@ export function separateDisplayNameAndHOCs(
 
   switch (type) {
     case ElementTypeClass:
-    case ElementTypeForwardRef:
```

Comment:
This condition branch was invalid anyway (do not used in `<Badge />`, `printElement`), so it doesn't matter that match does not performed now in the cases of `ElementTypeForwardRef` and `ElementTypeMemo`.